src/pages/edge/adobe-journey-optimizer/push-notification/push-token-sync-improvements.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Push Token Sync Improvements
+description: This guide provides information regarding push token sync optimizations made to prevent unnecessary network requests from being sent from your app.
+keywords:
+- Adobe Journey Optimizer
+- Messaging
+- Push Notification
+- iOS
+- Android
+
+---
+import Tabs from './tabs/push-token-sync-improvements.md'
+
+# Push Token Sync Improvements
+
+## Overview
+
+The AEPMessaging extension SDK prevents unnecessary network requests from being sent when syncing push tokens. By default, the push token sync optimization will only send a network request if the token has changed since the last succesful sync request. This reduces unnecessary network traffic which can help improve app performance.
+
+## Configuration
```

Comment:
nit: I'm currently working on the changes to the Tags Messaging extension to add a checkbox for this configuration. We don't need to hold up this PR, but once that change is made this section should be updated to mention the configuration can be changed remotely.